src/westpa/core/sim_manager.py

```diff
@@ -286,8 +286,18 @@ def initialize_simulation(self, basis_states, target_states, start_states, segs_
         # Process start states
         # Unlike the above, does not create an ibstate group.
         # TODO: Should it? I don't think so, if needed it can be traced back through basis_auxref
-        self.get_bstate_pcoords(start_states)
-        self.report_basis_states(start_states)
+
+        try:
+            if start_states[0].state_id is None:
+                last_id = basis_states[-1].state_id
+                for start_state in start_states:
+                    start_state.state_id = last_id + 1
+                    last_id += 1
+        except IndexError:
```

Comment:
It looks like this IndexError would only ever be raised if the start states are empty, yeah? Or if basis_states is of length 0. If that's accurate, let's just handle that with the logic (i.e. `if .... and len(start_states) > 0:`)

Reply:
If you do it as `if len(start_states) > 0 and start_states[0].state_id is None:` (i.e. put the len() check first), it'll short circuit if that first check isn't passed, and will only attempt the second if the length is greater than zero

Reply:
Thanks for the tip! It works and is fixed!